src/js/common/utils/addressFunctions.js

```diff
@@ -88,6 +88,18 @@ export function convertStateCodeFilterToStateCode (stateCodeFilter) {
   return '';
 }
 
+export function isValidStateCode (incomingStateCode) {
+  if (incomingStateCode) {
+    // Log incoming State Code
+    console.log('incomingStateCode:', incomingStateCode);
```

Comment:
Please comment out console.log's like this in future commits.

Reply:
Will do! Sorry about that. I thought I got them all!